core/src/main/resources/org/jobrunr/dashboard/frontend/src/components/notifications/top-app-bar-notifications-center.js

```
@@ -0,0 +1,278 @@
+        case 'jobs-not-found':
+            return `${problem.type};${problem.createdAt}`;
+        case 'severe-jobrunr-exception':
+            return `${problem.type};${problem.createdAt}`;
+        case 'cpu-allocation-irregularity':
+            return `${problem.type};${problem.createdAt}`;
+        case 'poll-interval-in-seconds-is-too-small':
+            return `${problem.type};${problem.createdAt}`;
+        case 'new-jobrunr-version':
+            return `${problem.type};${problem.latestVersion}`;
+        default:
+            return `unknown;${Math.random().toString(36).substring(2, 10)}`;
+    }
+}
+
+const isNotDismissed = (key, value) => !localStorage.getItem(key)?.endsWith(value);
+
+// why memoized: component re-renders because of job stats changes in TopAppBar
+export const TopAppBarNotificationsCenter = React.memo(() => {
+    const [clusterProblems, setClusterProblems] = useState([]);
+    const [latestVersion, setLatestVersion] = useState();
+    const [apiNotification, setApiNotification] = useState();
+    const {version: currentVersion} = useContext(JobRunrInfoContext);
+    const [isOpen, setIsOpen] = useState(false);
+    const popperAnchorEl = useRef(null);
+    const {toggleStatus, markAllRead, isRead} = useReadNotifications();
+
+    const resetLatestVersion = useCallback(() => {
+        setLatestVersion(undefined);
+    }, []);
+
+    const resetApiNotification = useCallback(() => {
+        setApiNotification(undefined);
+    }, []);
+
+    const reloadProblems = useCallback(() => {
+        fetch(`/api/problems`).then(res => res.json())
+            .then(problems => setClusterProblems(problems))
+            .catch(e => console.error("Failed to reload problems", e))
+    }, []);
+
+    useEffect(() => {
+        Promise.all([
+            fetch("/api/problems").then(res => res.json()),
+            fetch("https://api.jobrunr.io/api/version/jobrunr/latest").then(res => res.json()).catch(() => undefined /* ignored */),
```

Just to be sure: I presume we ignore any errors here (was already there) in case people block the outgoing call and we don't want to make a fuss/notice?